src/print.js

```diff
@@ -1,42 +1,76 @@
-const todoList = document.querySelector('.todoContainer');
+// Select the todoContainer element from the DOM
+const $todos = document.querySelector('.todoContainer');
 
-const data = [
-  {
-    description: 'Meet with Ken by 10 am Today',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Grace Anniversary',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Visit mama',
-    completed: false,
-    index: 1,
-  },
-];
+// Export a Todos class as the default export
+export default class Todos {
+  // Define a constructor for the Todos class
+  constructor() {
+    // Retrieve the todoLis array from local storage or create an empty array if none exists
+    this.todoList = JSON.parse(localStorage.getItem('todoList')) || [];
 
-data.sort((a, b) => a.index - b.index);
+    // If the todoLis array is not empty, loop through each item and add it to the DOM
+    if (this.todoList.length > 0) {
+      this.todoList.forEach((todo) => {
+        // Use insertAdjacentHTML to add a new todo item to the DOM
+        $todos.insertAdjacentHTML(
+          'beforeend',
+          `<ul class="listcontainer ">
+            <div class="checBox">
+              <input type="checkbox" class="checkboxClass" id="myCheckbox">
+            </div>
+            <li class="list textTodo">${todo.list}</li>
+            <i class="fa-solid fa-ellipsis-vertical vertical" ></i>
+            <i class="fa-solid fa-trash-can delete" id="${todo.list}"></i>
+          </ul>
+          
+          `,
+        );
+        // Add a click event listener to the remove button for each todo item
+        const $removeButton = document.getElementById(`${todo.list}`);
+        if ($removeButton) {
+          $removeButton.addEventListener('click', () => {
+            this.remove(`${todo.list}`);
+          });
+        }
+      });
+    }
+  }
 
-let html = '';
+  // Define a remove method to remove a todo item from the DOM and the todoLis array
+  remove = (idSelected) => {
+    const { parentElement } = document.getElementById(idSelected);
+    if (parentElement) {
+      parentElement.remove();
+    }
```

Comment:
According to the project requirements, deleting a task should update the index property of the remaining items so they represent the current list order and are unique. Your tasks should have three properties (index, description, and completed). These properties should be updated when tasks are added and removed from the list. Please update your to-do list to implement these requirements🙏

<img width="764" alt="image" src="https://user-images.githubusercontent.com/37116322/230675699-4a7c9edb-d394-44b8-a787-d100be127b13.png">